pkgs/applications/misc/grip/default.nix

```diff
@@ -26,14 +21,9 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkg-config libtool ];
   buildInputs = [
     gtk2
-    glib
-    libgnome
-    libgnomeui
-    vte
     curl
     cdparanoia
     libid3tag
-    ncurses
```

Comment:
Does it run without ncurses? That's interesting. 

Usually ncurses is at least optional dependency, but you are removing it completely...

Reply:
Grip is a gtk2 application, I downloaded the tarball and worked through the `INSTALL`, which reads like so:

```
Requirements
============

To use Grip, you must have a few dependencies.
If you obtained Grip from an RPM or DEB file, then they are already installed.
The dependencies are:

A CD/DVD player (surprise, surprise)
Some GTK2 libraries.

On a Redhat/Fedora/Mageia/OpenMandriva or other RPM based distributions,
you need these packages installed:

gtk2-devel
cdda-devel
id3-devel


On Debian derivatives you need these packages, as listed in the Debian control file (as maintained by Josef Grosch):

libgtk2.0-dev
libcurl4-openssl-dev
libid3-3.8.3-dev


Installation
============

If you have the source distribution, you need to compile it.
To compile and install Grip, simply go through one of the following two steps after extracting the tarball:

Alternative 1, meson:

 * mkdir build
 * cd build
 * meson ../
 * ninja
 become root if you are installing in a non-user directory, then run
 * ninja install

Alternative 2, GNU autotools (The old and trusted way):

 * './configure'
 * 'make'
 become root if you are installing in a non-user directory, then run
 * 'make install'

For information on options to the configure process, run:

  ./configure --help

Grip adds the following options that can be specified to configure:

  --disable-shared-cdpar  -- This will force Grip to link statically, rather
                             than dynamically with cdparanoia

  --disable-cdpar         -- This disable linking with cdparanoia entirely

  --disable-shared-id3    -- This will force Grip to link statically, rather
                             than dynamically with id3lib

  --enable-werror         -- This will turn warnings into errors.
			     Use this option when creating and testing patches.

Note that if Gnome is installed in a non-standard place (such as /opt), you
may need to use the '--with-gnome' configure option for all libraries to be
found correctly.
```